pages/pipelines/hosted_agents/terminal_access.md

```diff
@@ -22,6 +22,19 @@ As a pipeline is being built, expand the relevant step and as its job is being b
 
 <%= image "terminal-button-on-job.png", alt: "Accessing the SSH button through the Buildkite UI" %>
 
+To extend the terminal session time, it is recommended to include a `sleep` [command](/docs/pipelines/command-step) within your job steps. This can help maintain an active terminal connection and prevent the session from timing out too quickly, allowing you to debug your job or investigate the environment the job is running in.
```

Comment:
Looks good to me @dimiexplorer !

Just one minor suggestion since you're referring to the reader in the 2nd person (for consistency)...

```suggestion
To extend the terminal session time, it is recommended that you include a `sleep` [command](/docs/pipelines/command-step) within your job steps. This can help maintain an active terminal connection and prevent the session from timing out too quickly, allowing you to debug your job or investigate the environment the job is running in.
```